zmon-controller-app/src/main/resources/config/application.yml

```diff
@@ -85,3 +85,12 @@ flyway:
   schemas:
     - "zzm_data"
     - "zzm_api"
+
+logging:
+  level:
+    com.lightstep: DEBUG
```

Comment:
We really don't want this in production. If we want this level of logging temporarily, we shouldn't enable it using other means